lib/common/options.js

```diff
@@ -185,6 +192,21 @@ Options.modes = {
   }
 }
 
+Options.safeKeys = [
```

Comment:
Is there any reason to use `safeKeys` too? I mean if modules trying to add additional options, it will be discared in `build.config.js`

Reply:
Actually, I added `safeKeys` firstly, then added `unsafeKeys`, maybe I should delete it, if so, we need to add additional config into `unsafeKeys` when  it includes some fs specified opreation, such as `__dirname`.

How do you think ?

Reply:
I will delete `safeKeys` and add some items to `unsafeKeys`